reference/5.1/PowershellGet/Find-Module.md

```diff
@@ -49,6 +48,12 @@ If the **RequiredVersion** parameter is specified, `Find-Module` only returns th
 that is an exact match to the specified version. `Find-Module` searches through all available
 modules, because name conflicts between sources can occur.
 
+The parameters that take module version number take strings formatted as version numbers.
+
+- Standard version numbers have a format of `x.y.z` where x, y, and z are numbers
```

Comment:
Does it not support the four-digit version schemes, like `1.2.3.4`? Or is that what's meant by "standard" here? Same comment for two-digit schemes.

Reply:
Don't know but not going to get into it here.